galgebra/printer.py

```diff
@@ -963,6 +1031,55 @@ def latex(expr, **settings):
     return GaLatexPrinter(settings).doprint(expr)
 
 
+def latex_print(*s, **kws):
+
+    s = list(s)
+
+    GaLatexPrinter.fmt_dict = {'t': False, 'h': False}
+
+    if isinstance(s[0], str):
+
+        if s[0] == 'h':
+            GaLatexPrinter.fmt_dict['h'] = True
+            s = s[1:]
```

Comment:
This looks like a bad idea to me. This changes the behavior of `print(my_str)` to print nothing at all if `my_str == 'h'`!